### PR TITLE
Virtual chart repo

### PR DESCRIPTION
**Pull request by shivani170**
# Description
Screenshot 2023-05-24 at 7 40 51 PM

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/v2/values/chartValuesDiff/ChartValuesView.component.tsx [hunk 1]
```diff
@@ -329,6 +330,11 @@ const renderValidationErrorLabel = (message?: string): JSX.Element => {
     )
 }
 
+const renderVirtualEnvironmentInfoText = (): JSX.Element => {
+  // return GeneratedHelmDownload && <GeneratedHelmDownload />
```
Comment (vivek-devtron) [inline src/components/v2/values/chartValuesDiff/ChartValuesView.component.tsx:334]: Remove commented code and format

File: src/components/v2/values/chartValuesDiff/ChartValuesView.tsx [hunk 2]
```diff
@@ -153,7 +154,7 @@ function ChartValuesView({
     const isUpdate = isExternalApp || (commonState.installedConfig?.environmentId && commonState.installedConfig.teamId)
     const validationRules = new ValidationRules()
     const [showUpdateAppModal, setShowUpdateAppModal] = useState(false)
-
+console.log(commonState)
```
Comment (vivek-devtron) [inline src/components/v2/values/chartValuesDiff/ChartValuesView.tsx:157]: Remove this line

File: src/components/v2/values/chartValuesDiff/ChartValuesView.tsx [hunk 3]
```diff
@@ -1437,6 +1450,9 @@ function ChartValuesView({
                                     linkText={renderConnectToHelmChart()}
                                 />
                             )}
+                            {
+                              isVirtualEnvironment  && renderGeneratedDownloadManifest()
```
Comment (vivek-devtron) [inline src/components/v2/values/chartValuesDiff/ChartValuesView.tsx:1454]: we should add isVirtualEnv check inside the renderGeneratedDownloadManifest method

**Review by vivek-devtron — APPROVED** [review]

File: src/components/v2/values/chartValuesDiff/ChartValuesView.tsx (hunk 3 above)
Comment (shivani170) [reply src/components/v2/values/chartValuesDiff/ChartValuesView.tsx]: Done

**Pull request merged**